recipes/example/meta.yaml

```diff
@@ -33,7 +33,7 @@ build:
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```

Comment:
Please don't modify this file.

Reply:
sorry for that, it was inadvertently formatted. I undid the changes.